api/app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers/getCategories.controller");
+
+app.get("/api/categories", getCategories);
+
+
+app.use((err, request, response, next) => {
```

Comment:
If all errors are handled, then we usually want to make sure to return a 500 Internal Server Error here in our app.use